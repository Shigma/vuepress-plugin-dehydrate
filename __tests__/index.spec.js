const { resolve } = require('path')
const { fs } = require('@vuepress/shared-utils')
const { createApp } = require('@vuepress/core')

describe('dehydation', () => {
  const app = createApp(resolve(__dirname, 'docs'))

  beforeAll(async () => {
    await app.process()
    await app.build()
  }, 60000)

  test('app', () => {
    expect(app.base).toBe('/')
  })
})
