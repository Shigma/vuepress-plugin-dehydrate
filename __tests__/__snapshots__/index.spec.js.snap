// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dehydation 404.html 1`] = `
<!DOCTYPE html>
<html lang="en-US">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>VuePress</title>
    <meta name="description" content="">
    <link rel="preload" href="/assets/css/0.styles.53e4595a.css" as="style">
    <link rel="preload" href="/assets/js/app.3fe70f2e.js" as="script">
    <link rel="preload" href="/assets/js/2.418462f5.js" as="script">
    <link rel="prefetch" href="/assets/js/3.db027a4f.js">
    <link rel="prefetch" href="/assets/js/4.f6ba1d08.js">
    <link rel="prefetch" href="/assets/js/5.222ba868.js">
    <link rel="prefetch" href="/assets/js/6.d7ad24ac.js">
    <link rel="stylesheet" href="/assets/css/0.styles.53e4595a.css">
  </head>

  <body>
    <div id="app"></div>
    <script src="/assets/js/app.3fe70f2e.js" defer></script>
    <script src="/assets/js/2.418462f5.js" defer></script>
  </body>

</html>
`;

exports[`dehydation index.html 1`] = `
<!DOCTYPE html>
<html lang="en-US">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>VuePress</title>
    <meta name="description" content="">
    <link rel="preload" href="/assets/css/0.styles.53e4595a.css" as="style">
    <link rel="preload" href="/assets/js/app.3fe70f2e.js" as="script">
    <link rel="preload" href="/assets/js/3.db027a4f.js" as="script">
    <link rel="preload" href="/assets/js/5.222ba868.js" as="script">
    <link rel="prefetch" href="/assets/js/2.418462f5.js">
    <link rel="prefetch" href="/assets/js/4.f6ba1d08.js">
    <link rel="prefetch" href="/assets/js/6.d7ad24ac.js">
    <link rel="stylesheet" href="/assets/css/0.styles.53e4595a.css">
  </head>

  <body>
    <div id="app">
      <div class="content default">
        <p>readme</p>
      </div>
      <div class="global-ui"></div>
    </div>
    <script src="/assets/js/app.3fe70f2e.js" defer></script>
    <script src="/assets/js/3.db027a4f.js" defer></script>
    <script src="/assets/js/5.222ba868.js" defer></script>
  </body>

</html>
`;

exports[`dehydation noscript.html 1`] = `
<!DOCTYPE html>
<html lang="en-US">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>VuePress</title>
    <meta name="description" content="">
    <link rel="stylesheet" href="/assets/css/0.styles.53e4595a.css">
  </head>

  <body>
    <div id="app">
      <div class="content default">
        <p>noscript</p>
      </div>
      <div class="global-ui"></div>
    </div>
  </body>

</html>
`;
